web/assets/auto-apply-filter.js

```diff
@@ -0,0 +1,11 @@
+export default function autoApplyFilter() {
+    const filters = document.querySelectorAll("input.govuk-checkboxes__input");
+
+    filters.forEach((filter) => {
+        filter.addEventListener('click', () => {
+            setTimeout(() => {
+                document.forms["search-filters"].submit();
```

Comment:
You could target the filter's closest form parent, which would avoid tightly-coupling to its ID:
```suggestion
        filter.addEventListener('click', () => {
            setTimeout(() => {
                filter.closest("form").submit();
```